src/ray/core_worker/task_interface.h

```diff
@@ -136,11 +137,9 @@ class CoreWorkerTaskInterface {
   std::vector<std::shared_ptr<raylet::TaskArgument>> BuildTaskArguments(
       const std::vector<TaskArg> &args);
 
-  /// Translate from WorkLanguage to Language type (required by taks spec).
-  ///
-  /// \param[in] language Language for a task.
-  /// \return Translated task language.
-  ::Language ToTaskLanguage(WorkerLanguage language);
+  /// All the task submission providers supported.
+  std::unordered_map<int, std::unique_ptr<CoreWorkerTaskSubmitter>>
+      task_submission_providers_;
```

Comment:
`task_submission_providers_` -> `task_submitters_`?
The above comment needs to be updated as well.